library/data/tags/build.gradle

```diff
@@ -6,16 +6,16 @@ qslModule {
 	moduleName = "tags"
 	version = "1.0.0"
 	library = "data"
-	coreDependencies([
-			"resource_loader"
-	])
-	coreTestmodDependencies([
-			"qsl_base",
-			"lifecycle_events"
-	])
-	interLibraryTestmodDependencies("command", [
-			"command"
-	])
+	moduleDependencies {
+		core {
+			api("resource_loader")
+			testmodOnly("qsl_base")
```

Comment:
This feels strange, might need to test again if it launches correctly without this specified.

Reply:
Not sure what you mean by "strange". Looking at the diff it should be fine. 

And testing breaks the tags API... :ohno:

Reply:
This behavior exists before this pr.

Reply:
I know, I put it there because for some reason qsl_base wasn't exposed to the tags API through transitivity using resource loader, this is why I say it's strange.
If that behavior remains, that's unfortunate.